src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bigImg {
+  margin-top: 25px;
+  margin-bottom: 26px;
+  margin-left: 25px;
+  height: 19px;
+  width: 19px;
+}
+
+.bigBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  margin-top: 20px;
+  height: 70px;
+}
+
+.bigBar input {
+  background: transparent;
+  flex: 1;
+  border: 0;
+  margin-left: 17px;
+  height: 22px;
+  outline: none;
+}
+
+.bigBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.smallImg {
+  height: 11px;
+  width: 11px;
+  margin-top: 15px;
+  margin-bottom: 16px;
+  margin-left: 13px;
+}
+
+.smallBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  height: 42px;
+  margin-top: 20px;
+}
+
+.smallBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smallBar input {
+  background: transparent;
+  font-family: "Avenir", sans-serif;
+  flex: 1;
+  border: 0;
+  outline: none;
+  height: 20px;
+  margin-left: 9px;
+}
+
+.inputSearchSmall::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
set it to the html selector, so other elements will inherit it

Reply:
done, i hope)

Reply:
not fixed. You should move it to html selector, not *

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/97547824/230760022-4787a55f-b0d6-47aa-8778-5f45f3ffd6b9.png)
Not fixed